src/sb_gazebo/robots/rover_2018.xacro

```diff
@@ -0,0 +1,287 @@
+    <primary_rocker_arm lr_prefix="L" fb_prefix="BACK" x_reflect="-1" y_reflect="1"/>
+    <primary_rocker_arm lr_prefix="R" fb_prefix="BACK" x_reflect="-1" y_reflect="-1"/>
+
+    <xacro:macro name="primary_wheel_axle" params="lr_prefix fb_prefix x_reflect y_reflect">
+        <joint name="${lr_prefix}_${fb_prefix}_primary_rocker_arm_to_primary_wheel_axle" type="fixed">
+            <parent link="${lr_prefix}_${fb_prefix}_primary_rocker_arm"/>
+            <child link="${lr_prefix}_${fb_prefix}_primary_wheel_axle"/>
+            <origin xyz="
+                ${primary_wheel_axle_x_offset*x_reflect}
+                ${primary_wheel_axle_y_offset*y_reflect}
+                ${primary_wheel_axle_z_offset}
+                " 
+                rpy="0 0 0"
+            />
+        </joint>
+        <cylinder_link 
+            name="${lr_prefix}_${fb_prefix}_primary_wheel_axle"
+            x_pos="0"
+            y_pos="0"
+            z_pos="0"
+            radius="${primary_wheel_axle_radius}"
+            length="${primary_wheel_axle_length}"
+            roll="${pi/2}" 
+            pitch="${pi/2}" 
+            yaw="0"
+            mass="0.05"
+            material_name="grey"
+            />
+    </xacro:macro>
+    <primary_wheel_axle lr_prefix="L" fb_prefix="FRONT" x_reflect="1" y_reflect="1"/>
+    <primary_wheel_axle lr_prefix="R" fb_prefix="FRONT" x_reflect="1" y_reflect="-1"/>
+    <primary_wheel_axle lr_prefix="L" fb_prefix="BACK" x_reflect="-1" y_reflect="1"/>
+    <primary_wheel_axle lr_prefix="R" fb_prefix="BACK" x_reflect="-1" y_reflect="-1"/>
+
+    <xacro:macro name="primary_wheel" params="lr_prefix fb_prefix x_reflect y_reflect">
+        <joint name="${lr_prefix}_${fb_prefix}_primary_wheel_axle_to_primary_wheel" type="continuous">
+            <parent link="${lr_prefix}_${fb_prefix}_primary_wheel_axle"/>
+            <child link="${lr_prefix}_${fb_prefix}_primary_wheel"/>
+            <origin xyz="
+                ${primary_wheel_x_offset*x_reflect}
+                ${primary_wheel_y_offset*y_reflect}
+                ${primary_wheel_z_offset}
+                " 
+                rpy="0 0 0"
+                />
+            <axis xyz="0 1 0" rpy="0 0 0" />
+            <limit effort="100" velocity="100"/>
+            <joint_properties damping="0.0" friction="0.0"/>
+            <hardwareInterface>EffortJointInterface</hardwareInterface>
+        </joint>
+        <cylinder_link 
+            name="${lr_prefix}_${fb_prefix}_primary_wheel"
+            x_pos="0" 
+            y_pos="0" 
+            z_pos="0" 
+            radius="${primary_wheel_radius}"
+            length="${primary_wheel_length}"
+            roll="${pi/2}" 
+            pitch="${pi/2}" 
+            yaw="0"
+            mass="1"
+            material_name="grey"
+            />
+    </xacro:macro>
+    <primary_wheel lr_prefix="L" fb_prefix="FRONT" x_reflect="1" y_reflect="1"/>
```

Comment:
Same as above, expanding to LEFT/RIGHT would make it easier to understand

Reply:
This is a great point in general. There are a few things here that are not named verbosely enough.

Changed to `left/right` to be more consistent with our naming scheme (lower snake case). 
Also changed `lr_prefix`->`left_right_prefix`, `fb_prefix`->`front_back_prefix`.